cmd/zetae2e/local/local.go

```diff
@@ -208,6 +208,8 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	if !skipSetup {
 		noError(deployerRunner.FundEmissionsPool())
 	}
+	// Run the proposals under the start sequence(proposals_e2e_start folder)
+	noError(deployerRunner.CreateGovProposals(runner.StartOfE2E))
```

Comment:
This needs to be moved after setup otherwise it will run twice (once during setup, then once during the actual run.